<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationWidget</class>
 <widget class="QWidget" name="TranslationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>153</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TranslationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_language">
        <property name="title">
         <string>Language</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="le_summary"/>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="tb_delete">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/pisido.qrc">
             <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="pte_description"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/pisido.qrc"/>
 </resources>
 <connections/>
</ui>
